tests/load-tests/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- perf-team
```

Comment:
why not just the reviewers, like other team's directories? is it because the perf team only has 2 members?

Reply:
Reason is that I think it's fair to let perf-team approve changes to their own test suite 🤷

Reply:
That makes sense to me, I asked because other services are marked as "reviewers" within their directories: [example](https://github.com/redhat-appstudio/e2e-tests/blob/main/tests/integration-service/OWNERS). So, should they be promoted to "approvers" to their dirs as well?

Reply:
I guess they could be promoted to approvers, although the idea in that case was that each of the services already have their own QE, who is listed in the root OWNERS file as an approver.

And that is not a case for perf team anymore, because Nafta left the team